optimum_benchmark/import_utils.py

```diff
@@ -0,0 +1,18 @@
+import importlib.util
+
+_torch_available = importlib.util.find_spec("torch") is not None
+_onnxruntime_available = importlib.util.find_spec("onnxruntime") is not None
+_is_openvino_available = importlib.util.find_spec("openvino") is not None
+_is_neural_compressor_available = importlib.util.find_spec("neural-compressor") is not None
```

Comment:
I think it should be "neural_compressor" (that's why its test is failing)

Reply:
Thanks!